casper/src/slowcooker/scala/coop.rchain.casper/HashSetCasperSpecification.scala

```diff
@@ -1,247 +1,247 @@
+//    override type Result = List[String]
+//
+//    override def run(sut: Sut): Result = {
+//      val validator    = sut.nodes(node.idx)
+//      val blockMessage = (deploy(validator.node, deployment(0)) >> create(validator.node)).result
+//      validator.update(Some(blockMessage))
+//      validator.node.logEff.errors ++ validator.node.logEff.warns
+//    }
+//
+//    override def nextState(state: State): State =
+//      state.patch(node.idx, Seq(node.copy(deployed = true)), 1)
+//
+//    override def preCondition(state: State): Boolean = !state(node.idx).deployed
+//
+//    override def postCondition(state: State, result: Try[Result]): Prop =
+//      result.isSuccess && result.get.isEmpty
+//
+//  }
+//
+//  def isValidAndNoErrors(result: Try[(ValidBlockProcessing, List[String])]): Boolean =
+//    result.isSuccess && result.get._1.isRight && result.get._2.isEmpty
+//
+//  case class Propose(node: RNode) extends Command {
+//    override type Result = (ValidBlockProcessing, List[String])
+//
+//    override def run(sut: Sut): Result = {
+//      val validator = sut.nodes(node.idx)
+//      val result    = add(validator.node, validator.lastBlock.get).result.right.get
+//      validator.update(None)
+//      (result, validator.node.logEff.errors ++ validator.node.logEff.warns)
+//    }
+//
+//    override def nextState(state: State): State =
+//      state.patch(node.idx, Seq(node.copy(deployed = false)), 1)
+//
+//    override def preCondition(state: State): Boolean = state(node.idx).deployed
+//
+//    override def postCondition(state: State, result: Try[Result]): Prop = isValidAndNoErrors(result)
+//  }
+//
+//  case class DeployAndPropose(node: RNode) extends Command {
+//    override type Result = (ValidBlockProcessing, List[String])
+//
+//    override def run(sut: Sut): Result = {
+//      val validator = sut.nodes(node.idx).node
+//      val result = (deploy(validator, deployment(0)) >> create(validator) >>= (
+//          b => add(validator, b)
+//      )).result.right.get
+//      (result, validator.logEff.errors ++ validator.logEff.warns)
+//    }
+//
+//    override def nextState(state: State): State = state
+//
+//    override def preCondition(state: State): Boolean = !state(node.idx).deployed
+//
+//    override def postCondition(state: State, result: Try[Result]): Prop = isValidAndNoErrors(result)
+//  }
+//
+//  case class Receive(node: RNode) extends Command {
+//    override type Result = List[String]
+//
+//    override def run(sut: Sut): List[String] = {
+//      val validator = sut.nodes(node.idx).node
+//      validator.receive().result
+//      validator.logEff.errors ++ validator.logEff.warns
+//    }
+//
+//    override def nextState(state: State): State = state
+//
+//    override def preCondition(state: State): Boolean = true
+//
+//    override def postCondition(state: State, result: Try[Result]): Prop =
+//      result.isSuccess && result.get.isEmpty
+//  }
+//}
```

Comment:
What is the status of this thing?

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-798109618